geopandas/tests/test_geodataframe.py

```diff
@@ -594,15 +594,15 @@ def test_from_features_unaligned_properties(self):
         p3 = Point(3, 3)
         f3 = {
             "type": "Feature",
-            "properties": {"a": 2},
+            "properties": None,
             "geometry": p3.__geo_interface__,
```

Comment:
Can you keep this as it is (including the change below) and add a new test using the snippet with `null` you posted in #2242?

Reply:
I've added a new test, let me know if it's fine!
Cheers